Automattic/newspack-plugin
# Add completion handling to AdSense wizard and add Google Analytics wizard

### All Submissions:






### Changes proposed in this Pull Request:



Closes #135  .

This PR has a few related changes. The changes are as follows:
- Removed duplicate/dummy `class-sitekit-configuration-manager.php` and updated things to use the real configuration manager (`class-site-kit-configuration-manager.php`).
- Added a feature when the user comes to the AdSense wizard that will check if AdSense if configured, and if it is configured, it will render that AdSense setup is complete and mark it complete on the checklist. (screenshot below)
- Implemented a Google Analytics wizard based on the AdSense wizard. It has basically the same UI as the AdSense Wizard. It's linked directly from the Analytics dashboard card. This is slightly different than what the designs called for, and we can update it when we add another supported analytics provider. When the wizard is complete, you should see the Analytics card marked as 'complete' (screenshot below)

### Screenshots
![localhost_wp-admin_admin php_page=newspack-google-adsense-wizard (3)](https://user-images.githubusercontent.com/7317227/60539454-9ee5eb00-9cc1-11e9-9c61-320d597f58cb.png)
![localhost_wp-admin_admin php_page=newspack-google-adsense-wizard (2)](https://user-images.githubusercontent.com/7317227/60539190-f6d02200-9cc0-11e9-86ac-6130544fc502.png)
![localhost_wp-admin_admin php_page=newspack (1)](https://user-images.githubusercontent.com/7317227/60539201-f899e580-9cc0-11e9-8009-605dd61fecdd.png)

### How to test the changes in this Pull Request:

If you have Site Kit working on your local environment, you should be able to build things like normal and simply test the 2 wizards. 

If you don't have Site Kit working on your local environment, you can change [this line](https://github.com/Automattic/newspack-plugin/blob/master/includes/configuration_managers/class-site-kit-configuration-manager.php#L62) to `return true;` to simulate that a module has been successfully configured.

### Other information:
